### DISCUSSION TITLE
参考文献使用“著录-出版年制”时，附录的引用显示有问题。

### DISCUSSION THREAD

**Discussion by ghost** [tuna/thuthesis#971] [Q&A] [answered]
按照说明文件在thusetup.tex中进行了修改：“
% 著者-出版年制
\usepackage{natbib}
\bibliographystyle{thuthesis-author-year}
”
但是发现附录的参考文献格式引用处有问题，具体如图所示，变成了上标，且括号的格式变了。正文中没有这样的问题。编译方法是用的GNU Make，安装的是TexLive 2023。

![屏幕截图 2024-07-15 111331](https://github.com/user-attachments/assets/5d9e74ca-9298-4efe-95dc-15a80716e2ff)

**Comment by zepinglee** [comment]
我能复现这个 bug。大致是因为 <https://github.com/tuna/thuthesis/issues/837> 中的修改没考虑到著者-出版年制的情况。

**Reply by ghost** [reply]
大神好，我比较小白，刚开始学着使用~想问下大神有没有什么比较方便快速的解决方法呢？

**Reply by zepinglee** [reply]
> 大神好，我比较小白，刚开始学着使用~想问下大神有没有什么比较方便快速的解决方法呢？

比较复杂，我晚上处理一下。

**Reply by zepinglee** [reply, marked as answer]
我看了一下，模板本身没有问题，正常写作即可。出错原因在于示例文档为了展示效果，在 `chap04.tex` 将引用样式切换为顺序编码制，导致附录中的格式也出错。只要删掉 `chap04.tex` 的相关内容就恢复正常。

https://github.com/tuna/thuthesis/blob/60661f7d140c192418cc9151b54a6c59a99fae23/data/chap04.tex#L62-L64

**Reply by ghost** [reply]
已解决，感谢大神！
